source/new-starter/access/index.html.md.erb

```diff
@@ -69,7 +69,7 @@ or [raise an IT request](https://gdshelpdesk.digital.cabinet-office.gov.uk/helpd
 
 The Account team use [Seal Slack bot](https://github.com/alphagov/seal) to give helpful prompts on Slack. One example is the morning reminder for pull requests that require review.
 
-Seal needs to know which GitHub user is in which team. This configuration is described in the [Seal alphagov.yml file](https://github.com/alphagov/seal/blob/master/config/alphagov.yml#L128-L162).
+Seal needs to know which GitHub user is in which team. This configuration is described in the [Seal alphagov.yml file](https://github.com/alphagov/seal/blob/master/config/alphagov.yml#L151-L181).
```

Comment:
I think just remove the line references, as otherwise we'll need to change these almost every time any GOV.UK team changes.